src/excmds.ts

```diff
@@ -4409,4 +4409,13 @@ browser.runtime.onInstalled.addListener(details => {
     // could add elif "update" and show a changelog. Hide it behind a setting to make it less annoying?
 })
 
+/** Opens optionsUrl for the selected extension in a new tab.
+ *
+ * NB: Tridactyl cannot run on this page!
+ */
+//#content
+export async function extoptions(optionsUrl: string) {
+    return tabopen(optionsUrl)
```

Comment:
I think I would prefer it if the clever bit of matching the extension name happened here so it could be useful as a non-interactive command. What do you think?

Also it might make more sense to open a `winopen -popup` if Tridactyl can't run on these pages, but I'm not 100% sure.

Reply:
> I think I would prefer it if the clever bit of matching the extension name happened here so it could be useful as a non-interactive command.

Do you mean to say that `extoptions` should receive an extension object and then get the values of its members? 

> Also it might make more sense to open a winopen -popup if Tridactyl can't run on these pages, but I'm not 100% sure.

Sure, we can do that if you think it makes more sense that way.

Reply:
E.g. extoptions("ublock") should open the ublock extensions page. If extension names are often not unique then we should allow the URL approach too, but I think that would clutter up command history a little much.

If the extension names are almost always unique then the completion should insert the name rather than the URL.

Reply:
Got this working. Should I make the options page open in a popup window?

Reply:
Could you give it a go and see what feels more natural to you?

I won't have time to actually test this code for quite a while by the way.

Reply:
I think a popup window feels more natural. I'll make the options open in a popup. Thank you.